src/lib/components/Hamburger.svelte

```diff
@@ -0,0 +1,69 @@
+<script lang="ts">
+	import HamburgerIcon from './icons/HamburgerIcon.svelte';
+	import CloseIcon from './icons/CloseIcon.svelte';
+
+	import SaveIcon from './icons/SaveIcon.svelte';
+	import ScoreSheetIcon from './icons/ScoreSheetIcon.svelte';
+	import EndGameIcon from './icons/EndGameIcon.svelte';
+	import SettingsIcon from './icons/SettingsIcon.svelte';
+
+	import { fade } from 'svelte/transition';
+	import statefulFade from '$lib/stateful-fade';
+
+	const { state, fadeTo, fadeOut } = statefulFade('close');
+
+	import type { ComponentType } from 'svelte';
+
+	type OptionTitle = 'Score Sheet' | 'Save Game' | 'Settings' | 'End Game';
+	type Option = { title: OptionTitle; icon: ComponentType };
+
+	const hamburgerOptions: Option[] = [
+		{ title: 'Score Sheet', icon: ScoreSheetIcon },
+		{ title: 'Save Game', icon: SaveIcon },
+		{ title: 'Settings', icon: SettingsIcon },
+		{ title: 'End Game', icon: EndGameIcon }
+	];
+
+	function handleHamburger() {
+		if ($state === 'open') {
+			fadeTo('close');
+		} else if ($state === 'close') {
+			fadeTo('open');
+		}
+	}
```

Comment:
since we're using separate buttons
```suggestion
	function handleHamburger() {
		fadeTo('close');
        }
        function handleClose() {
		fadeTo('open');	
	}
```